AppBuilder/platform/dataFields/ABFieldList.js

```diff
@@ -458,35 +11,22 @@ module.exports = class ABFieldList extends ABFieldListCore {
       this._Selectivity = new ABFieldSelectivity(values, object);
    }
 
-   /*
-    * @function propertiesComponent
-    *
-    * return a UI Component that contains the property definitions for this Field.
-    *
-    * @param {App} App the UI App instance passed around the Components.
-    * @param {stirng} idBase
-    * @return {Component}
-    */
-   static propertiesComponent(App, idBase) {
-      return ABFieldListComponent.component(App, idBase);
-   }
-
    ///
    /// Instance Methods
    ///
 
-   save() {
+   async save() {
```

Comment:
let's not declare this `async` unless we rewrite the `save()` to use `await` in the asynchronous code.